conf/messages.cy

```diff
@@ -171,4 +172,11 @@ claimPeriod.Oct2021 = October 2021
 claimPeriod.Nov2021 = November 2021
 claimPeriod.Dec2021 = December 2021
 claimPeriod.error.required = Select one value
-claimPeriod.error.invalid = Enter a real claim period start
\ No newline at end of file
+claimPeriod.error.invalid = Enter a real claim period start
+
+
+startPage.title = Coronavirus Job Support Scheme calculator
```

Comment:
Capitalise first letter of calculator - Ramon has it capitalised in this wireframes

Reply:
fixed